java/backup-restore/src/main/java/com/instaclustr/cassandra/backup/cli/CommitLogBackupApplication.java

```diff
@@ -0,0 +1,41 @@
+package com.instaclustr.cassandra.backup.cli;
+
+import static com.instaclustr.cassandra.backup.cli.BackupRestoreCLI.init;
+import static com.instaclustr.picocli.CLIApplication.execute;
+
+import com.google.inject.Inject;
+import com.instaclustr.cassandra.backup.cli.BackupRestoreCLI.CLIJarManifestVersionProvider;
+import com.instaclustr.cassandra.backup.impl.commitlog.BackupCommitLogsOperationRequest;
+import com.instaclustr.sidecar.operations.OperationsService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "commitlog-backup",
+        mixinStandardHelpOptions = true,
+        description = "Upload archived commit logs to remote storage.",
+        sortOptions = false,
+        versionProvider = CLIJarManifestVersionProvider.class
+)
+public class CommitLogBackupApplication implements Runnable {
```

Comment:
Does this now support restoring commitlog segments?

Reply:
I am not so far with this but yes, that is the goal ...